x-pack/plugins/endpoint/public/applications/endpoint/store/alerts/types.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { AlertData } from '../../../../../endpoint_app_types';
+
+export interface AlertListState {
+  alerts: AlertData[];
```

Comment:
might be good to use `readonly alerts: readonly AlertData[];` if you intend to memoize things using `reselect` or `useMemo` or if you intend to pass values as props to `React.memo`'d components.